lib/access-code-manager.js

```diff
@@ -0,0 +1,69 @@
+"use strict";
+
+module.exports = (config, promise, random, storageServiceFactory) => {
+    var codeLength, secretLength, storageService;
+
+    /**
+     * Create a set of API credentials
+     *
+     * @param {string} accountId
+     * @param {accessCodeManager~createRequest} request
```

Comment:
Great catch!  I remember distinctly thinking that I need to define this as I typed the jsdoc, but I didn't make a note of it so I promptly forgot.